### ISSUE TITLE
About the ResNet backbone

### ISSUE THREAD

**Issue by lxysl** [hongxin001/JoCoR#16]
Dear Author:
Your work is awesome and it inspired me. 
I want to try it using the ResNet backbone, and there are several problems for me.
The following experiments are executed on cifar-10, noise ratio=0.5, and co_lambda is 0.5.

* the loss and accuracy
I test the CNN+adam and ResNet (random init+sgd/adam, and finetune the cls head+sgd).
I found the loss on ResNet (upper three) is higher than CNN (the lowest one), and the accuracy is lower.

loss:
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/45818075/203327844-eeff3d31-7866-4906-835e-b84cc9ff59e7.png">
accuracy:
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/45818075/203328365-075cf387-e46e-4707-9015-fe5db04c0b80.png">

* the confidence
The confidence here means the probability of the predict class on the train set, and I plot the 75th, 50th, 25th 
 percentile confidence as follows.
I found the confidence of ResNet far lower than CNN.
The confidence of CNN is close to 100%, while that of ResNet is about 40%.
<img width="1037" alt="image" src="https://user-images.githubusercontent.com/45818075/203329260-d291934e-fa34-443a-9fca-294f5e8e7b1e.png">

I wonder what's the reason of that?
Have you observed this odd phenomenon?
Or if you know any researches on this topic?
I'm really looking forward to your reply! Many thanks!
